encoder.lua

```diff
@@ -173,12 +173,24 @@ this.create_clip = function(clip_type, on_complete)
 end
 
 this.set_encoder_alive = function()
+    local args_mpvnet = { 'mpvnet', '--version' }
+    local process_result_mpvnet = function(_, ret, _)
+        --  for some reason stdout is empty
+        if ret.status ~= 0 then
+            this.alive = false
```

Comment:
`this.alive` isn't used in the code to test anything. I guess it could be used to panic if alive is false, but as of now you can remove it.

Reply:
encoder.alive is used in videoclip.lua line 259